### ISSUE TITLE
Schemas to be defined for Skill, Employment, and education entities 

### ISSUE THREAD

**Issue by sujitrs** [beckn/DSEP-Specification#1]
Schemas to be defined for Skill, Employment, and Education registry entities so that they become discoverable using Beckn :
1. Learner
2. Employer
3. Trainer
4. Assessor
5. Assessment Agency
6. Content Curator
7. Content
8. Courses
9. Skill Hubs
10. Scholarship
11. Internship
12. Loans

**Comment by sujitrs** [comment]
Below diagram may help to imagine what we intend to achieve:
![Discover _X_ (1)](https://user-images.githubusercontent.com/23429265/146630440-51e0d6b3-6cee-44f6-b4cf-ff14b929ac28.jpg)

**Comment by core-wg-admin** [comment]
@sujitrs some of the participants may play dual roles and might not be as straight forward as the above diagram depicts. 

Some of them may play the role of both BAPs and BPPs leading to a cascaded / parallel network. I suggest re-modelling the above network to a cascaded / parallel network based on use cases instead of roles. 

What is the supply chain here?

There might be content creators who may be the original source of information. Content curators may consume the content as BAPs but may create courses that are published as catalogs thereby acting as BPPs. Students may consume the courses via BAPs but might might upload their course completion certificates to a skill registry as BPPs. 

In conclusion, I suggest creating separate network models that fulfill a particular set of use cases and then connect them together in various topologies to fulfill various use cases. What do you think?

**Comment by core-wg-admin** [comment]
@sujitrs please take a look at [THIS]( https://github.com/beckn/DSEP-Specification/blob/documentation/docs/protocol-drafts/DSEP-001-DSEP-Open-Network-Architecture-Reference-Draft-01.md) document. It shold provide a clearer representation of the architecture that will emerge after implementing the specification.

**Comment by sukhpreetsamagra** [comment]
@sujitrs @core-wg-admin - At Samagra we had created a schema for education considering whatever exists within the government, Sunbird and NDEAR. Please refer to it here - http://139.59.25.99/

We can use this as a starting point.

**Comment by parthlawate** [comment]
@sukhpreetsamagra Is there an updated link for the Education Schema ?

**Comment by sukhpreetsamagra** [comment]
You can use this. 
https://shikshaplatform.io/
